frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -649,22 +714,29 @@ const MonitoringDashboardsPage: React.FC<MonitoringDashboardsPageProps> = ({ mat
         });
         dispatch(monitoringDashboardsPatchAllVariables(allVariables));
 
-        // Set time range options to their defaults since they may have been changed on the
-        // previous dashboard
-        dispatch(monitoringDashboardsSetEndTime(null));
-        dispatch(monitoringDashboardsSetTimespan(MONITORING_DASHBOARDS_DEFAULT_TIMESPAN));
+        // Set time range and poll interval options to their defaults or from the query params if available
+
+        refreshInterval &&
```

Comment:
We should avoid using the ternary operator for control flow here too, but can handle with a follow up PR.